Samples/WindowsCopilotRuntime/cs-wpf-sparse/WCRforWPF/AppxManifest.xml

```diff
@@ -0,0 +1,58 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package 
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:uap2="http://schemas.microsoft.com/appx/manifest/uap/windows10/2"
+  xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10"
+  xmlns:uap10="http://schemas.microsoft.com/appx/manifest/uap/windows10/10"
+  IgnorableNamespaces="uap uap2 uap3 rescap desktop uap10">
+  <Identity
+    Name="WCRforWPFSparse"
+    ProcessorArchitecture="x64"
+    Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
```

Comment:
>CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US [](http://example.com/codeflow?start=15&length=80)

Simplify. The MAUI sample uses: `CN=User Name`

Reply:
That may or may not need to match the certificate used in the .ps1, but this could resolve my comment there as well.

Reply:
Yea I will need to update the PS1. I am wondering if this will change the AUMID too. I will find out